src/tests/replace/replace-test.cu

```diff
@@ -0,0 +1,143 @@
+/*
```

Comment:
Should add more documentation/comments to the tests

Reply:
done

Reply:
@nsakharnykh Can you approve your review since this is resolved?